openfpga_flow/tasks/fpga_bitstream/generate_bitstream/configuration_chain/device_4x4_PathBitSetting/config/bitstream_annotation.xml

```diff
@@ -0,0 +1,9 @@
+<!--
+////////////////////////////////////////////////////////////////////////////////
+# THIS FILE WAS AUTOMATICALLY GENERATED FROM DO NOT EDIT
+////////////////////////////////////////////////////////////////////////////////
+-->
+<openfpga_bitstream_setting>
+  <bit value="0" path="fpga_top.grid_io_left_0__1_.logical_tile_io_mode_io__7.logical_tile_io_mode_physical__iopad_0.GPIO_DFFR_mem[0]"/>
```

Comment:
Consider to group the bit overwrite to a node, which is easy to manage.

```
<openfpga_bitstream_setting>
  <bitstream_overwrite>
    <bit value="0" path="fpga_top.grid_io_left_0__1_.logical_tile_io_mode_io__7.logical_tile_io_mode_physical__iopad_0.GPIO_DFFR_mem[0]"/>
  </bitstream_overwrite>
</openfpga_bitstream_setting>
```

Reply:
Group everything under <bitstream_overwrite>. This is done via 
https://github.com/chungshien/OpenFPGA/commit/933155b08fe9e6b3d2d5bb3133ffbbfb0b5822dd (code) and https://github.com/chungshien/OpenFPGA/commit/cbe9a46f95094dd3f0b6842685f9a83c8d4e08a6 (doc)